jni/src/test/java/com/horizen/sigproofnative/NaiveThresholdSigProofTest.java

```diff
@@ -1,314 +1,314 @@
+//                86, -38, 28, 5, 100, 84, -65, 0, 0, -88, 77, -35, -113, 61, -14, 99, -11, -78, -106, 76, -88, -14, -99,
+//                -64, -47, 115, -56, -6, 99, -4, 24, -98, 86, 91, 65, -117, -14, -85, -22, 104, -87, -85, -66, -107, -35,
+//                -23, 114, 7, -2, -107, -80, 47, 33, 53, 102, 12, -40, 110, 41, 3, 45, 57, 29, -117, -67, -42, -114, -98,
+//                -95, -37, -18, 71, -38, -30, 62, 73, -66, 62, 115, -79, 18, -102, -62, 9, -62, 54, 29, 58, -58, -44, 20,
+//                -46, -12, -106, 61, -18, -107, 48, -43, 98, -1, 33, -21, 0, 0
+//            },
+//            {
+//                38, -26, -94, -117, -34, -114, 42, 37, 41, 47, 95, -16, -123, -57, 71, 127, 90, 31, -116, 104, 51, -77,
+//                -74, 61, 38, -63, -26, -1, 88, 126, 103, 26, 50, -13, 40, 53, -8, -30, 70, 80, -17, 63, -28, -107, -92,
+//                38, -61, 111, -44, 74, 41, 40, -11, -77, -22, -69, -56, -82, 79, 33, 75, 114, 15, -26, -117, -92, 29,
+//                20, 105, -96, -69, 103, 82, 89, -128, -10, -103, -5, 21, -60, -32, 117, 114, -32, -127, -74, 106, -101,
+//                -80, 70, 2, 1, -63, 21, 0, 0, 40, 121, -85, -103, 85, -100, -116, -70, 114, 113, -128, 52, 105, -101,
+//                60, 113, 63, -91, -82, 24, -63, -84, -105, -36, -107, -1, -102, 5, -67, 28, -55, -39, -109, 23, 38, -9,
+//                74, -113, -49, 84, 117, -102, -16, 48, -86, 92, -60, 13, -67, 89, 109, -19, -105, 40, -114, 91, -36,
+//                -109, -22, 20, 109, -114, -112, 107, -60, -69, -3, 74, 113, -53, -93, -54, 36, -102, -123, 81, -8, -54,
+//                -28, 80, 99, -51, 71, 66, 113, -104, -60, -75, 114, -22, 22, -95, 82, -90, 0, 0
+//            }
+//        };
+//
+//        // Deserialize secret keys and get the corresponding public keys
+//        for (int i = 0; i<keyCount; i++) {
+//
+//            try(SchnorrSecretKey sk = SchnorrSecretKey.deserialize(secretKeyList[i]))
+//            {
+//                assertNotNull("sk" + i + "deserialization must not fail", sk);
+//
+//                SchnorrPublicKey pk = new SchnorrKeyPair(sk).getPublicKey();
+//                assertTrue("Public key verification failed.", pk.verifyKey());
+//
+//                publicKeyList.add(pk);
+//            }
+//        }
+//
+//        // Deserialize Schnorr Signatures
+//        for (int i = 0; i<keyCount; i++) {
+//            if (i < threshold) {
+//                SchnorrSignature sig = SchnorrSignature.deserialize(serializedSignatureList[i]);
+//                assertNotNull("sig" + i + "deserialization must not fail", sig);
+//                signatureList.add(sig);
+//            } else {
+//                SchnorrSignature sig = new SchnorrSignature();
+//                signatureList.add(sig);
+//            }
+//        }
+//
+//        createAndVerifyProof();
+//    }
+//
+//    @Test
+//    public void testCreateRandomProof() throws Exception {
```

Comment:
Don't see any usage of hardcoded data here. Maybe we can keep it ?

Reply:
This test depends on "samples" stored in jni/src/test/resources. Can I regenerate them somehow ?

Reply:
Yes, the test at https://github.com/HorizenOfficial/zendoo-sc-cryptolib/blob/updatable_poseidon_tweedle/api/src/ginger_calls.rs#L680 should generate and write to file the pk/vk/proof that I remember can be used for this test

Reply:
Ok, thank you, but sample_pk now has 1.2 GB size, and sample_vk 130 MB. Is it Ok to keep them in GIT?

Reply:
I pushed a fix in algebra_ocl_serializations yesterday, now the sizes should be much smaller. I suggest (temporarily) to change the code of function generate_params (see related comment)

Reply:
This will allow to upload them to git

Reply:
Hm, In GIT there is a limit for max file size: 100 MB. And current PK is 346 MB =(

Reply:
Let's generate them directly Java side, execute the test and then delete them.

Reply:
Done